conf/mesos-site.xml

```diff
@@ -53,6 +53,16 @@
   </property>
 
   <property>
+    <name>mesos.hdfs.mesosdns</name>
```

Comment:
It's for usability. I originally had it the other way around, but having an empty string denote functionality is really confusing (and this way follows hadoop convention)